ui/app/AppLayouts/Profile/LeftTab/components/MenuButton.qml

```diff
@@ -19,17 +20,28 @@ Rectangle {
     width: parent.width
     radius: Style.current.radius
 
-    Image {
+    Item {
         id: iconImage
-        source: menuButton.source
         height: 24
         width: 24
-        sourceSize.width: width
-        sourceSize.height: height
         anchors.left: parent.left
         anchors.leftMargin: Style.current.padding
         anchors.verticalCenter: parent.verticalCenter
-        fillMode: Image.PreserveAspectFit
+
+        Image {
+            id: iconSrc
+            source: menuButton.source
+            sourceSize: Qt.size(parent.width, parent.height)
```

Comment:
Just out of curiosity:

Any reason you've moved the `Image` here and wrapped it in an `Item`?
I assume it's for the usage of `Qt.size()` here? 

Wonder if we could get away without doing that, or maybe there's an advantage of doing this that I'm not aware of..

Reply:
I used ```sourceSize``` attribute instead of ```sourceSize.width``` and ```sourceSize.height```, it is same effect but more simple.
And then, The ```Image``` object doesn't have ```color``` attribute and needs to use ```ColorOverlay``` in order to change its color.
So, I needed to build a new ```Item``` has ```Image``` and ```ColorOverlay```.
Make sense?

Reply:
You can put the `ColorOverlay` inside the `Image` and then do 
```
anchors.fill: parent
source: parent
 ```
So you don't need the wrapping `Item`

Reply:
Sure, yours is the more compact way.
I just pushed new updates.